oxide-auth/src/code_grant/refresh.rs

```diff
@@ -532,6 +532,11 @@ impl Error {
 }
 
 impl ErrorDescription {
+    /// Create a new description from an access token error
+    pub fn new(error: AccessTokenError) -> Self {
+        Self { error }
+    }
```

Comment:
Being critical towards making too many internals public applies here, too, the constructor should only be needed by the response template internally. There's already constructors applicable to `new_ok` and `new_unauthorized` for instance.